includes/admin/dashboard-widgets.php

```diff
@@ -42,6 +42,33 @@ function edd_dashboard_sales_widget( ) {
 	echo '<p><img src=" ' . esc_attr( set_url_scheme( EDD_PLUGIN_URL . 'assets/images/loading.gif', 'relative' ) ) . '"/></p>';
 }
 
+/**
+ * Get the sales earnings/count data for the dashboard widget.
+ *
+ * @since 3.0.0
+ * @return array
+ */
+function edd_get_dashboard_sales_widget_data() {
+	$data   = array();
+	$ranges = array( 'this_month', 'last_month', 'today', 'total' );
+	foreach ( $ranges as $range ) {
+		$args = array(
+			'range'  => $range,
+			'output' => 'formatted',
+		);
+		if ( 'total' === $range ) {
+			unset( $args['range'] );
+		}
+		$sales          = new EDD\Stats( $args );
```

Comment:
Minor, but let's call this `$stats` since we have access to the full `EDD\Stats` and not just sales data here.